api/HTMLInputElement.json

```
@@ -2201,6 +2201,57 @@
           }
         }
       },
+      "showPicker": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLInputElement/showPicker",
+          "spec_url": "https://html.spec.whatwg.org/multipage/input.html#dom-input-showpicker",
+          "support": {
+            "chrome": {
+              "version_added": "99"
+            },
+            "chrome_android": {
+              "version_added": "99"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": false,
+              "notes": "Mozilla folks <a href='https://github.com/whatwg/html/pull/7319#issuecomment-988837778'>expressed interest</a>."
+            },
+            "firefox_android": {
+              "version_added": false,
+              "notes": "Mozilla folks <a href='https://github.com/whatwg/html/pull/7319#issuecomment-988837778'>expressed interest</a>."
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": false
```

Per https://groups.google.com/a/chromium.org/g/blink-dev/c/YfnM0ubs53k/m/LLhpJMihDQAJ this will be supported in WebView too:

```suggestion
              "version_added": "99"
```